MonitoringSetUp.sh

```diff
@@ -285,6 +285,7 @@ MyContainerId="$(sudo docker run -it --privileged --rm -d --network host --name
     
     sudo docker cp /tmp/collectd $MyContainerId:/etc/default/collectd
     sudo docker cp /tmp/mdsd $MyContainerId:/etc/default/mdsd
+    sudo docker exec  $MyContainerId  bash -c 'source /root/EnvVariables.sh;  ./RunMonAgents/RunMonAgents.sh >> /tmp/crontab.logs'
```

Comment:
Will you be making any change in the image? if yes then we can rename the log file here and in crontab.